.gitignore

```diff
@@ -1,4 +1,5 @@
 *.ipynb_checkpoints/
 *.zhistory
 Manifest.toml
-*.jl~
\ No newline at end of file
+*.jl~
+misc/Playground.ipynb
```

Comment:
Out of curiosity, what is ``misc/Playground.ipynb``? Is it just some scratch space?

Reply:
I foolishly replied to this in my email, see below